package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "0.0.96",
+  "version": "0.0.97",
```

Comment:
Strange.. I did not change that file it must have been due to this merge https://github.com/CJSCommonPlatform/govuk_single_page_pdk/pull/78/commits/9a0a6cb7b3738064865b6cf11153846789a40dd4